python_scripts/01_tabular_data_exploration.py

```diff
@@ -364,5 +364,5 @@
 #   and evaluating your machine learning model;
```

Comment:
=> "you may need to be careful when interpreting the values of performance metrics: for instance, the accuracy of a baseline prediction rule that constantly predicts the majority class can be much larger than 50%."